File: app/src/main/java/hu/napirajz/android/response/NapirajzData.java
package hu.napirajz.android.response;

import com.google.gson.annotations.SerializedName;

import java.util.Date;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class NapirajzData {

    @SerializedName("ID")
    private String id;
    @SerializedName("Cim")
    private String cim;
//    @SerializedName("Datum")
//    private Date datum;
    @SerializedName("URL")
    private String url;
    @SerializedName("LapURL")
    private String lapUrl;
    @SerializedName("Parbeszed")
    private String parbeszed;
    @SerializedName("Egyeb")
    private String egyeb;
}

File: app/src/main/kotlin/hu/napirajz/android/response/NapirajzData.kt
package hu.napirajz.android.response

import com.squareup.moshi.Json
import java.io.Serializable
import java.time.LocalDate

data class NapirajzData(
        @Json(name = "ID")
        val id: String,
        @Json(name = "Cim")
        val cim: String,
        @Json(name = "Datum")
        val datum: LocalDate,
        @Json(name = "URL")
        val url: String,
        @Json(name = "LapURL")
        val lapUrl: String?,
        @Json(name = "Parbeszed")
        val parbeszed: String,
        @Json(name = "Egyeb")
        val egyeb: String
) : Serializable
